doc/torque_api.md

```diff
@@ -0,0 +1,170 @@
+
+### Time Methods
+
+Method | Options | Returns | Description |
+---|---|---|---|
+`setStep(step)` | `time numeric` | `this` | the value must be between 0 and the total number of `steps` in the animation
+`play()` | | `this` | starts the animation
+`stop()` | | `this` | stops the animation and set time to step 0
+`pause()` | | `this` | stops the animation but keep the current time (play enables the animation again)
+`toggle()` | | `this` | toggles (pause/play) the animation
+`getStep()` | | current animation step (integer) | gets the current animation step. A step is considered an animation frame
+`getTime()` | | current animation time (Date) | gets the real animation time
+`isRunning()` | | `true`/`false` | describes whether the Torque layer is playing or is stopped
+
+**Note:** Torque.js interprets the beginning and ending date/time from your "Time Column" as one block, then divides that up into [Steps](/cartodb-platform/cartocss/properties-for-torque/#torque-frame-count-number), depending on the number you set. It does not necessarily draw one frame for each row. 
+
+### Layer Control Methods
+
+ Method | Options | Returns | Description
+---|---|---|---
+`hide()` | none | `this` | hides the Torque layer
+`show()` | none| `this` | shows the Torque layer
+
+### Style Methods 
+
+Method | Options | Returns | Description
+---|---|---|---|
+`setCartoCSS(cartocss)` | `cartocss string` | `this` | style the map rendering using client-side CartoCSS (not available with [Named maps](/cartodb-platform/maps-api/named-maps/))
+
+Torque supports a limited subset of CartoCSS parameters, as described in the [torque-reference](https://github.com/cartodb/torque-reference). Alternatively, Mapnik supports the following [mapnik-reference](https://github.com/cartodb/torque-reference) for Torque rendering. 
+To see the full list of supported parameters, read the [Torque CartoCSS documentation](/cartodb-platform/cartocss/properties-for-torque/). `value` and `zoom` variables can be used. `value` is the value of aggregation (see `countby` constructor option). `zoom` is the current zoom being rendered.
+
+TorqueLayer currently expects `marker` styling.
+
+#### Example
+
+This CartoCSS example should be `string` encoded in Javascript.
+
+```scss
+#layer {
+  marker-fill: #662506;
+  marker-width: 20;
+  [value > 1] { marker-fill: #FEE391; }
+  [value > 2] { marker-fill: #FEC44F; }
+  [value > 3] { marker-fill: #FE9929; }
+  [value > 4] { marker-fill: #EC7014; }
+  [value > 5] { marker-fill: #CC4C02; }
+  [value > 6] { marker-fill: #993404; }
+  [value > 7] { marker-fill: #662506; }
+  [frame-offset = 1] {  marker-width: 20; marker-fill-opacity: 0.05;}' // renders the previous frame
+  [frame-offset = 2] {  marker-fill: red; marker-width: 30; marker-fill-opacity: 0.02;}' // renders two frames ago from the current being rendered
+}
+```
```

Comment:
This is not a valid torque CartoCSS example. It's missing all required Map properties